### PR TITLE
Ui separation

### PR DESCRIPTION
**Pull request by violaviolet**
- files and directories update
- refactor new_base.txt, file path after changing files struture
- add_new_user divided into: get_data_from user and add_to_file
- add new user to file 
- separate printing functions
- separate getting data from user functions
- divide two functions search_user_by_name/ surname into displaying user and returning a list of searching user
- remove common code from search_user_by_name/ surname -> create new function search_user
- implementing enum class Search into search_users

### CODE REVIEW

File: phonebook/main_functions.py [hunk 1]
```diff
@@ -16,55 +23,61 @@ def load_data_from_file(users, filepath):
...
     phone_number = input("Enter user phone number: ")
+    user = {
+        "name": name,
+        "surname": surname,
+        "phone_number": phone_number,
+    }
+    return user
+
+
+def add_new_user(users, user_as_dictionary, filepath):
```
Comment (sparkzi) [inline phonebook/main_functions.py:38]: filepath is not used in this function

File: phonebook/main_functions.py [hunk 2]
```diff
@@ -16,55 +23,61 @@ def load_data_from_file(users, filepath):
...
-    searching_filter = input("Enter searching filter:")
+def show_users(users):
     for user in users:
-        print(user)
-"""
+        print(user.to_string())
 
 
-def search_user_by_name(users):
+def get_searching_name():
```
Comment (sparkzi) [inline phonebook/main_functions.py:57]: get_searching_name is not  named consequently ( get_data_from_user )

File: phonebook/main_functions.py (hunk 1 above)
Comment (violaviolet) [reply phonebook/main_functions.py:38]: filepath will be deleted

File: phonebook/main_functions.py [hunk 3]
```diff
@@ -16,55 +23,61 @@ def load_data_from_file(users, filepath):
...
+    if field == Search.NAME:
+        for user in users:
+            if user.name == user_data:
+                searched_users.append(user)
+        return searched_users
+    elif field == Search.SURNAME:
+        for user in users:
+            if user.surname == user_data:
+                searched_users.append(user)
+        return searched_users
```
[the remark this answers is not in the document]
Comment (violaviolet) [reply phonebook/main_functions.py:78]: correction was made

File: phonebook/main_functions.py (hunk 2 above)
Comment (violaviolet) [reply phonebook/main_functions.py:57]: changed into : get_data_to_create_new_user
get_data_to_search_by_name/surname

File: phonebook/main_functions.py [hunk 4]
```diff
@@ -1,6 +1,12 @@
 from User import User
-import data
 from datetime import datetime
+from enum import Enum
+
+
+class Search(Enum):
+    NAME = 1
+    SURNAME = 2
```
Comment (sparkzi) [inline phonebook/main_functions.py:8]: surename

**Pull request merged**